### ISSUE TITLE
Exonum doesn't accept messages larger than 100kb

### ISSUE THREAD

**Issue by GeneralDarky** [exonum/exonum#1464]
I modified timestamping example (rust service and node.js light client) to store in blockchain not only file hash, but whole base64 encoded file as string field.  It works fine until I try to send file larger than 100kb. In this case, error 413 pops up. Is there a way to avoid this limitation?

Ubuntu 18.04
Exonum 0.11.0 
Rust 1.36.0

**Comment by ivan-ochc** [comment]
Already fixed in v0.12.1

**Comment by aleksuss** [comment]
@GeneralDarky Exonum 0.11.1 has been released with a fix as well.

**Comment by GeneralDarky** [comment]
Still problem. Placing files larger 100 KB works fine now, but for files smaller 800 KB only. If you try to place a file larger than 800 KB sometimes you get network error (EPIPE) sometimes error code 500 or error code 413.

`[HPM] Proxy created: /  ->  http://127.0.0.1:8220
[HPM] Error occurred while trying to proxy request /api/explorer/v1/transactions from 127.0.0.1:2269 to http://127.0.0.1:8220 (EPIPE) (https://nodejs.org/api/errors.html#errors_common_system_errors)
`

**Comment by aleksuss** [comment]
@GeneralDarky have you tried to increase consensus config value: `max_message_len` ?

**Comment by GeneralDarky** [comment]
It helped, thanks! But max value must be less then 10485760.

`error: called `Result::unwrap()` on an `Err` value: Error("max_message_len (134217728) must be less then 10485760", line: 0, column: 0)`

Do you plan to increase the maximum message size?

**Comment by vitvakatu** [comment]
Reopening, as Exonum now does not support messages with a size of more than 10 Mb :D

**Comment by ivan-ochc** [comment]
@GeneralDarky, Does your service require messages with more than 10mb length?

**Comment by GeneralDarky** [comment]
> @GeneralDarky, Does your service require messages with more than 10mb length?

Yes, I'm going to store files in blockchain. Maximum size of 30-50 MB would be optimal.
